docs/reference/commandline/swarm_init.md

```diff
@@ -50,6 +55,14 @@ $ docker swarm init --listen-addr 192.168.99.121:2377 --auto-accept worker
 Swarm initialized: current node (1m8cdsylxbf3lk8qriqt07hx1) is now a manager.
 ```
 
+### `--cert-expiry`
+
+This flag sets the validity period for node certificates.
+
+### `--dispatcher-heartbeat`
+
+This flags sets the frequency with which manager will ask node for their health.
```

Comment:
I think nodes announce their presence to a manager periodically rather than being asked by the manager. @stevvooe can confirm.